### ISSUE TITLE
.stylelintignore support

### ISSUE THREAD

**Issue by row1** [shinnn/vscode-stylelint#178]
The extension is reporting errors in my SVGs which I want to ignore (not sure why it is looking at SVGs to start with).

I have tried to create a `.stylelintignore` file containing:
```
*.svg
```
But it doesn't seem to be respecting it.

**Comment by shinnn** [comment]
Exact duplicate of #167.
